iotedgehubdev/cli.py

```diff
@@ -70,8 +70,9 @@ def _parse_config_json():
             connection_str = config_json[CONN_STR]
             cert_path = config_json[CERT_PATH]
             gatewayhost = config_json[GATEWAY_HOST]
+            hub_conn_str = config_json.get(HUB_CONN_STR)
```

Comment:
can we sue config_json[HUB_CONN_STR]?

Reply:
No. get will return None if the key is not defined. But if use [], exception will be thrown